stable/jenkins/values.yaml

```diff
@@ -443,11 +443,15 @@ agent:
 
   # Executed command when side container gets started
   command:
+  # Default args value is "${computer.jnlpmac} ${computer.name}" but JCasC requires
+  # escaping with ^$ so the value is hardcoded in the templates and applied conditionally.
   args:
```

Comment:
You are trying to explain these code snippets here, right?

- ./templates/_helpers.tpl

  ```
          {{- if .Values.agent.args }}
          args: "{{ .Values.agent.args }}"
          {{- else }}
          args: "^${computer.jnlpmac} ^${computer.name}"
          {{- end }}
  ```

- ./templates/config.yaml
  ```
  {{- if .Values.agent.args }}
                    <args>{{ .Values.agent.args }}</args>
  {{- else }}
                    <args>${computer.jnlpmac} ${computer.name}</args>
  {{- end }}
  ```

You are absolutely right that this is confusing. If we have an empty value here then we still set a default by using if conditions. 

It would be best if we could set this here to:
```yaml
args: "${computer.jnlpmac} ${computer.name}"
```

and take care of escaping dollar signs in the template.
I moved that part into a separate PR https://github.com/helm/charts/pull/21773

Reply:
Nice! I left a minor comment on the PR.